### DISCUSSION TITLE
Welcome to react-step-builder Discussions!

### DISCUSSION THREAD

**Discussion by sametweb** [sametweb/react-step-builder#23] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BootprojectStore** [comment]
Hi and thank's for a really nice component.
I wonder if there's a way of keep track on the current step. If you for eg. update the page, then forward application to that step.

Any thoughts?

Regards Rickard

**Comment by sametweb** [comment]
Hello @BootprojectStore,

I recently published a major version of the library. This was something I had in mind. With your question, it became clearer. I will make sure to add a feature that enables this behavior. Thanks for the question and I apologize for the late reply.
